core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestion.java

```diff
@@ -90,30 +114,51 @@ protected boolean lessThan(Entry.Option a, Entry.Option b) {
         }
     }
 
-    @Override
-    public Suggest.Suggestion<Entry> reduce(List<Suggest.Suggestion<Entry>> toReduce) {
-        if (toReduce.size() == 1) {
-            return toReduce.get(0);
+    public static CompletionSuggestion reduceTo(List<Suggest.Suggestion> toReduce) {
```

Comment:
I am not sure we do have that but if not can we have a test for this method as well as javadocs?